### ISSUE TITLE
The Wordcount crash if the wordcount container is not defined

### ISSUE THREAD

**Issue by ma2ciek** [ckeditor/ckeditor5#2575]
Right now the logic is as follows:

```js
// constructor()
this._config = editor.config.get( 'wordCount' ) || {};
```

```js
// init()
if ( isElement( this._config.container ) ) {
	this._config.container.appendChild( this.wordCountContainer );
}
```

```js
// destroy()
destroy() {
	this._outputView.element.remove();
	this._outputView.destroy();

	super.destroy();
}
```

```js
get wordCountContainer() {
	// ...

	if ( !this._outputView ) {
		this._outputView = new View();
	}

	// ...
}
```

So the `this._outputView` might be undefined in the `destroy()` method and this will end up with the `Cannot read property 'element' of undefined` error.

Ref: https://github.com/ckeditor/ckeditor5-angular/issues/123.

**Comment by devmnnit** [comment]
> @msamsel hi, I want to know if this bug has fixed in @ckeditor/ckeditor5-word-count v10.0.1? I met the same problem in angular8 when destroy ckeditor5 component?

I am facing problem to implement wordcount plugin in custom ckeditor5 build in angular. It is throwing **ckeditor-duplicated-modules**

Can you please help me to resolve this ?

**Comment by ma2ciek** [comment]
> I am facing problem to implement wordcount plugin in custom ckeditor5 build in angular. It is throwing **ckeditor-duplicated-modules**

This seems to be a problem with the installation or using this plugin. I suspect that you are adding a source plugin to the build, instead of [rebuilding the build with the added plugin](https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/installing-plugins.html#adding-a-plugin-to-a-build).
